src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.cs

```diff
@@ -333,22 +326,8 @@ private static int BinarySearch(ImmutableArray<Node> nodes, string name)
         /// </summary>
         private static readonly ConditionalWeakTable<MetadataId, AsyncLazy<SymbolTreeInfo>> s_metadataIdToInfo = new();
 
-        private static Task<SpellChecker> GetSpellCheckerAsync(
-            SolutionServices services, SolutionKey solutionKey, Checksum checksum, string filePath, ImmutableArray<Node> sortedNodes)
-        {
-            // Create a new task to attempt to load or create the spell checker for this 
-            // SymbolTreeInfo.  This way the SymbolTreeInfo will be ready immediately
-            // for non-fuzzy searches, and soon afterwards it will be able to perform
-            // fuzzy searches as well.
-            return Task.Run(() => LoadOrCreateSpellCheckerAsync(services, solutionKey, checksum, filePath, sortedNodes));
-        }
-
-        private static Task<SpellChecker> CreateSpellCheckerAsync(
-            Checksum checksum, ImmutableArray<Node> sortedNodes)
-        {
-            return Task.FromResult(new SpellChecker(
-                checksum, sortedNodes.Select(n => n.Name.AsMemory())));
-        }
+        private static SpellChecker CreateSpellChecker(Checksum checksum, ImmutableArray<Node> sortedNodes)
```

Comment:
is this really needed anymore?

Reply:
this is actually called by a few places :)  so i'dlike to leave in.